update/files/multiFileReader.go

```diff
@@ -19,6 +20,7 @@ type multiFileReader struct {
 	importFolder string
 	files        map[string]io.Closer
```

Comment:
There are no mutexes used when working with all these three maps in this file: files, dataReader and filenames

Reply:
I know, they work in a serial manner. Won't add mutex as the whole package will be removed in the next PR.

Reply:
it was used in a serial way. so no need for mutexes.

Reply:
ok